lab1/print_style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  padding: 50px 100px;
+  background-color: white;
+  font-family: "Comic Sans MS", arial;
+  font-size: 100%;
+  text-align: justify;
+  text-justify: inter-word;
+}
+nav {
+  display:none;
```

Comment:
will do, because we have to make it look like a professional printed document where a navigation bar can't exist